ui/tests/layout-and-ui-smoke.test.tsx

```
@@ -0,0 +1,451 @@
+    render(
+      <MemoryRouter>
+        <ThemeProvider>
+          <LanguageProvider>
+            <Navbar />
+          </LanguageProvider>
+        </ThemeProvider>
+      </MemoryRouter>,
+    );
+
+    fireEvent.mouseDown(screen.getAllByRole('button', { name: /Theme|Tema/i })[0]);
+    screen.getAllByRole('button', { name: /Light|Luminos/i }).forEach((button) => fireEvent.click(button));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)